Makefile

```diff
@@ -22,38 +22,46 @@ export GOSUMDB ?= sum.golang.org
 GIT_COMMIT  = $(shell git rev-list -1 HEAD)
 GIT_VERSION = $(shell git describe --always --abbrev=7 --dirty)
 # By default, disable CGO_ENABLED. See the details on https://golang.org/cmd/cgo
-CGO         ?= 0
+CGO  ?= 0
+PWD  := $(shell pwd)
 
 LOCAL_ARCH := $(shell uname -m)
 ifeq ($(LOCAL_ARCH),x86_64)
-	TARGET_ARCH_LOCAL=amd64
+  TARGET_ARCH_LOCAL=amd64
 else ifeq ($(shell echo $(LOCAL_ARCH) | head -c 5),armv8)
-	TARGET_ARCH_LOCAL=arm64
+  TARGET_ARCH_LOCAL=arm64
 else ifeq ($(shell echo $(LOCAL_ARCH) | head -c 4),armv)
-	TARGET_ARCH_LOCAL=arm
+  TARGET_ARCH_LOCAL=arm
 else
-	TARGET_ARCH_LOCAL=amd64
+  TARGET_ARCH_LOCAL=amd64
 endif
 export GOARCH ?= $(TARGET_ARCH_LOCAL)
 
 LOCAL_OS := $(shell uname)
 ifeq ($(LOCAL_OS),Linux)
-   TARGET_OS_LOCAL = linux
+  TARGET_OS_LOCAL = linux
 else ifeq ($(LOCAL_OS),Darwin)
-   TARGET_OS_LOCAL = darwin
+  TARGET_OS_LOCAL = darwin
 else
-   TARGET_OS_LOCAL ?= windows
+  TARGET_OS_LOCAL ?= windows
 endif
 export GOOS ?= $(TARGET_OS_LOCAL)
 
-ifeq ($(GOOS),windows)
-BINARY_EXT_LOCAL:=.exe
-GOLANGCI_LINT:=golangci-lint.exe
-# Workaround for https://github.com/golang/go/issues/40795
-BUILDMODE:=-buildmode=exe
+# If docker is installed, use that for compatability with Github CI
```

Comment:
We also need to add a check to ensure we do not use Docker when running in Github Workflows! There we manually install the correct version of Golangci-lint.

Reply:
can you add a check that ensures the environment variable `CI` is not true, and only in that case run this Docker step?

https://docs.github.com/en/actions/learn-github-actions/environment-variables

Reply:
Yeah this makes sense to check the version rather than use Docker. I will need to figure out how to specify which version of the linter to use

Reply:
The version should be detected autosomally.

Reply:
Sorry - can you explain what you mean by "autosomally?"

Reply:
Just typo, detect version info from `GOLANGCILINT_VER` in `dapr.yml`

Reply:
Should be fixed in latest update to the Makefile